CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 - Add `enrollments` value to `TargetingAttributes` — it is a set of strings containing all enrollments, past and present ([#5685](https://github.com/mozilla/application-services/pull/5685)).
   - _Note: This change only applies to stateful uses of the Nimbus SDK, e.g. mobile_
 - Add ability to enroll selected features multiple times (coenrollment) ([#5684](https://github.com/mozilla/application-services/pull/5684), [#5697](https://github.com/mozilla/application-services/pull/5697)).
+- Update enrollment logic to re-evaluate enrollment for rollouts ([#5716](https://github.com/mozilla/application-services/pull/5716)).
```

Comment:
Could you add a sentence and PR link to https://github.com/mozilla/application-services/pull/5716/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR24